sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/BlobHelper.java

```diff
@@ -0,0 +1,70 @@
+package com.azure.storage.blob.implementation.util;
+
+import com.azure.storage.blob.BlobServiceVersion;
+import com.azure.storage.blob.implementation.AzureBlobStorageImpl;
+import com.azure.storage.blob.models.CpkInfo;
+import com.azure.storage.blob.specialized.BlobAsyncClientBase;
+
+public class BlobHelper {
```

Comment:
Should this be named `AsyncBlobHelper` to be consistent with naming? The sync version is called `SyncBlobHelper`.

Or `BlobHelperSync` and `BlobHelperAsync`.

Reply:
I'd say it's a BlobClientHelper, not just blobs. . . 🤔